src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Columns/Content/cart_status_data.html.twig

```diff
@@ -0,0 +1,38 @@
+{#**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+{# No longer used - which results in different behavior than in legacy controller #}
```

Comment:
I dont understand this comment 😅 if it's not longer used, why not delete it ?

Reply:
Not sure if it will pass the QA without this template.  
![image](https://user-images.githubusercontent.com/32385991/73729860-797cfc00-473e-11ea-8a29-4c4b397d592b.png)
Legacy one: 
![image](https://user-images.githubusercontent.com/32385991/73729894-87328180-473e-11ea-976d-6fcf3928acc6.png)
Looks different without it; Also for some reason sorting is not working the same in the migrated page for Order ID column

Reply:
Resolved this one - confusion comes from that OrderId grid column uses the data from 'status' field in the db. So at some point I have changed the name from status to order_id which led to this template not being used. Now reverted the order_id to be status again. And then this template is used and badges are displayed as expected.